src/Apps/MyCollection/Routes/EditArtwork/Utils/createOrUpdateArtwork.ts

```diff
@@ -0,0 +1,47 @@
+import { MyCollectionCreateArtworkInput } from "__generated__/useCreateArtworkMutation.graphql"
+import { MyCollectionUpdateArtworkInput } from "__generated__/useUpdateArtworkMutation.graphql"
+import { useCreateArtwork } from "../Mutations/useCreateArtwork"
+import { useUpdateArtwork } from "../Mutations/useUpdateArtwork"
+
+export type ArtworkInput =
+  | MyCollectionCreateArtworkInput
+  | MyCollectionUpdateArtworkInput
+
+export const useCreateOrUpdateArtwork = () => {
```

Comment:
It would be good to name this file after the primary export `useCreateOrUpdateArtwork`. That way next dev who comes along will a) know that its a hook; and b) know that they can't import it into anywhere but another hook or a react component lest things error

Reply:
Thanks, I've missed renaming the file 👍